services/generator/service_test.go

```diff
@@ -1442,23 +1442,6 @@ func TestGenerateTemplateData(t *testing.T) {
 		assert.Equal(t, "/api/", templateData.IngressPath)
 	})
 
-	t.Run("AppendSlashStarToIngressPathIfUseGCEIngressIsTrue", func(t *testing.T) {
```

Comment:
Within the helm chart for `estafette-ci` I ran into the issue that GCE ingress still needs `pathType: ImplementationSpecific` and have paths end in `*`. Is that different now? Newer version of GCE ingress controller that supports `pathType: Prefix`?

Reply:
See related issue https://github.com/cockroachdb/helm-charts/pull/171/files